symfit/contrib/interactive_guess/interactive_guess.py

```diff
@@ -8,62 +8,86 @@
 
 import matplotlib.pyplot as plt
 import numpy as np
+from scipy.stats import gaussian_kde
 
 plt.ioff()
 
 #from pkg_resources import parse_version
 
-#SUPPORTED_VERSION = '0.3.0'
+#SUPPORTED_VERSION = '0.3.6'
 #SYMFIT_VERSION = symfit.version  # something something.
 #
 #if parse_version(SUPPORTED_VERSION) != parse_version(SYMFIT_VERSION):
 #    raise EnvironmentError("Your symfit version is not supported. YMMV.")
 
 
-class InteractiveGuess2D(TakesData):
+class InteractiveGuess(TakesData):
     """A class that provides an graphical, interactive way of guessing initial
     fitting parameters."""
 
-    @keywordonly(n_points=100)
+    @keywordonly(n_points=50, log_contour=True, percentile=[5, 95])
```

Comment:
[5, 95] should be a tuple or None. http://python-guide-pt-br.readthedocs.io/en/latest/writing/gotchas/?highlight=default%20